html_utils/utils.py

```diff
@@ -0,0 +1,21 @@
+from bs4 import BeautifulSoup
+
+def get_title(soup: BeautifulSoup):
+    # return the title of the page of empty string if not found
+    title = soup.find("title")
+    return title.string if title else None
+
+def get_meta_name(soup: BeautifulSoup):
+    # finds the meta tag and returns the content of the name attribute or empty string if not found
+    meta_name = soup.find("meta", {"name": "description"})
+    return meta_name["content"] if meta_name else None
```

Comment:
what's the default value when `soup.find()` doesn't find anything? If it's None, you don't need to add the `if` statement in line 11.

Reply:
.find() returns either:
Tag | NavigableString | None (when does not find nothing)

I need the if there to check if meta name will have the methods to search the key of interest.

I will add a more readable alternative

Reply:
alternative added